site/pt/blog/new-in-devtools-111/index.md

```diff
@@ -113,39 +135,50 @@ Chromium issues: [1407586](https://crbug.com/1407586), [1402891](https://crbug.c
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/0a7c428980b8f8a60e83c780e17f6a6f94007493 #} 
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/aea51aad7026881ad26ca1b270d12338adb92d79 #} 
 
-Chromium issue: [1339771](https://crbug.com/bbb)
+Issue relacionada: [1339771](https://crbug.com/bbb)
 
 
 <!-- ## Better syntax highlight for Angular {: #syntax } -->
+## Melhor syntax highlight para Angular {: #syntax }
```

Comment:
nit: "Syntax highlight melhor para Angular"

Reply:
Hmm, I'm not sure, I think this option makes the text a bit more informal, can you provide a 3rd input @alvarocamillont?

Reply:
I wouldn't block on this discussion (nit is for nitpicking)